src/pages/Admin/Venue/Rooms/RoomsForm.tsx

```diff
@@ -320,29 +321,19 @@ const RoomInnerForm: React.FC<RoomInnerFormProps> = (props) => {
                   </div>
                   <div className="toggle-room">
                     <div className="input-title">Enabled ?</div>
```

Comment:
Assuming we use the `label` prop on the `Toggler`, this will be redundant:
```suggestion
```

Reply:
Idea I had tried beforehand and noticed styling issues that may become usability ones I deemed OOS to this PR.

With `label` prop:
![sparkle-toggles-02](https://user-images.githubusercontent.com/79229621/125162100-66b5f480-e186-11eb-8e32-156c8e323b59.png)

Original:
![sparkle-toggles-03](https://user-images.githubusercontent.com/79229621/125162099-661d5e00-e186-11eb-8b4a-d91feab18ee5.png)

Reply:
Can you have a quick look and see what the reasoning for that is. If it's only a couple of little SCSS style changes to tweak it to work well here it would help accessibility, so would be good to do. If it'll take ages or be too big a change then we can OOS it, but for now, worth checking.

Reply:
My primary concern is restoring basic usability, even if suboptimal. So I'm selectively accepting those suggested changes first. Leaving styling issues for last. Will comment more on this later.